test/TestUtilities/Test.Utility/Commands/TestRestoreRequest.cs

```diff
@@ -79,7 +79,8 @@ public class TestRestoreRequest : RestoreRequest
                     log: log),
                 cacheContext,
                 clientPolicyContext,
-                log)
```

Comment:
Can you add unit tests that utilize your new caching capability for correctness check?